dependencies/pom.xml

```diff
@@ -127,13 +124,14 @@
         <version.lib.narayana>5.12.0.Final</version.lib.narayana>
         <version.lib.netty>4.1.100.Final</version.lib.netty>
         <version.lib.netty-io_uring>0.0.19.Final</version.lib.netty-io_uring>
-        <version.lib.oci>2.60.1</version.lib.oci>
+        <version.lib.oci>2.66.0</version.lib.oci>
```

Comment:
If you wanted, you could also remove the by-value import of the contents of their BOM, since it seems to now be valid in version 2.66.0.

Reply:
Hmmm...does not look like it is fixed in 2.66.0? https://github.com/oracle/oci-java-sdk/blob/8d42b2916aeeb3b27384ec8d6eac348f9eaaf170/bmc-bom/pom.xml#L10

Reply:
Ah, you are right; missed the parent block